src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -625,12 +624,11 @@ else if (world.getEnvironment() == Environment.NORMAL)
         this.config_blockSurfaceOtherExplosions = config.getBoolean("GriefPrevention.BlockSurfaceOtherExplosions", true);
         this.config_blockSkyTrees = config.getBoolean("GriefPrevention.LimitSkyTrees", true);
         this.config_limitTreeGrowth = config.getBoolean("GriefPrevention.LimitTreeGrowth", false);
-        this.config_checkPistonMovement = config.getBoolean("GriefPrevention.CheckPistonMovement", true);
-        this.config_pistonsInClaimsOnly = config.getBoolean("GriefPrevention.LimitPistonsToLandClaims", true);
-        if (!this.config_checkPistonMovement && this.config_pistonsInClaimsOnly) {
-            AddLogEntry("Error: You have enabled LimitPistonsToLandClaims, but CheckPistonMovement is off!");
-            this.config_pistonsInClaimsOnly = false;
-        }
+        this.config_pistonMovement = PistonMode.of(config.getString("GriefPrevention.PistonMovement", "CLAIMS_ONLY"));
+        if (config.isBoolean("GriefPrevention.LimitPistonsToLandClaims") && !config.isBoolean("GriefPrevention.LimitPistonsToLandClaims"))
```

Comment:
Looks like a typo. You have isBoolean twice.

Reply:
Good catch, thanks!